plugins/trace_api_plugin/include/eosio/trace_api_plugin/chain_extraction.hpp

```diff
@@ -19,8 +20,8 @@ class chain_extraction_impl_type {
     * @param except_handler called on exceptions, logging if any is left to the user
     * @param shutdown called when this class determines application should shutdown because of fatal error
     */
-   chain_extraction_impl_type( StoreProvider& store, std::function<void(std::exception_ptr)> except_handler )
-   : store(store)
+   chain_extraction_impl_type( StoreProvider store, std::function<void(std::exception_ptr)> except_handler )
+   : store(std::move(store))
```

Comment:
As this is a templated parameter, I think it should be `StoreProvider&& store` and `std::forward<StoreProvider>(store)`.  I've made the same situation in my code come to think of it 😇

Reply:
Not for class template type. If the constructor was templated then yes.

Reply:
right... :). ☕️